pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -2066,6 +2068,38 @@ func (gce *GCECloud) List(filter string) ([]string, error) {
 	return instances, nil
 }
 
+// GetAllZones returns all the zones in which nodes are running
+func (gce *GCECloud) GetAllZones() (sets.String, error) {
```

Comment:
I'm guessing it's easier to do this than make it a private method in the same module, like we did for aws ?